include/aws/common/array_list.inl

```diff
@@ -91,7 +91,7 @@ bool aws_array_list_is_valid(const struct aws_array_list *AWS_RESTRICT list) {
 
 AWS_STATIC_IMPL
 void aws_array_list_clean_up(struct aws_array_list *AWS_RESTRICT list) {
-    AWS_PRECONDITION(aws_array_list_is_valid(list));
+    AWS_PRECONDITION(aws_array_list_is_valid(list), "Input array_list [list] should be valid.");
```

Comment:
I wonder if "must" is better than "should" here.

Reply:
I think I agree. "Should" sounds less strict than "must".